### PR TITLE
New nav structure

### PR DESCRIPTION
**Pull request by vgantchev**
New nav structure

### CODE REVIEW

**Comment by vgantchev** [comment]
@jak-pan @SimonKraus

**Comment by SimonKraus** [comment]
i'm okay with this approach. 
Only thing is i would name the  "Snakenet" category "How-to" because most articles are  still valid after snakenet. 
Maybe putting snakenetas  subsection of how-to.
We should do this  quick as  we have many translations ready to be published

**Comment by jak-pan** [comment]
Yeah nominating and validating guides will pass to mainnet. I'd not get them under a subsection if we're changing stuff. Maybe just make snakenet a single article under learn?

**Comment by SimonKraus** [comment]
@vgantchev i talked to jakub and we agreed not to nest the documents in the filesystem for various reasons.
we do not want to deal with redirects
we need to be able to reorganize nav without messing around with redirects and file locations

so the documents will stay in the single directory (don't worry, we'll find what to edit).

We just edit the nav structure.

So we can do the following categories in the main nav:
"How-to" with claim, become a nominator, become a validator, connect to a site
"Learn" with Staking and Snakenet
"Maintain" with node-setup, performance bench, node monitoring
"Build" <= can be added later when content is available

@jak-pan please take a look once again and confirm :pray:

**Comment by jak-pan** [comment]
Yeah, the flat structure has multiple positives.
- It's easier to i18n
- If you update menu structure because it's no longer relevant (which happens a lot), you don't need to change all of the links and update every file, including i18n.
- You will never need to make redirects
- It's harder to duplicate stuff in a large knowledge base, everything is in the same place.

Now it does have some disadvantages but it's easy to overcome them by having proper on-point names for the files, using edit button to find the doc you're searching for, or using search of the editor you are using.

**Comment by SimonKraus** [comment]
@vgantchev  let me know if i should apply the new nav structure

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 .docusaurus
-build
```
Comment (jak-pan) [inline .gitignore:4]: build/

(don't want to commit build files)

**Comment by vgantchev** [comment]
Thanks for the suggestions guys! I changed the navs and removed the file tree. For now I think we can have Snakenet under Intro, as both are not very informative (i will pick them up soon though)

File: docs/build.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+id: build
```
Comment (jak-pan) [inline docs/build.md:2]: Probably don't need this unless we have something?

File: .gitignore (hunk 1 above)
Comment (vgantchev) [reply .gitignore:4]: yeah you are right, I will rename build.md

**Review by jak-pan — APPROVED** [review]

**Pull request merged**